.travis.yml

```diff
@@ -8,11 +8,11 @@ php:
   - nightly
 
 env:
-  - SYMFONY_VERSION=4.3.* SOLR_VERSION=8.1.1 SOLR_CLOUD=true
-  - SYMFONY_VERSION=4.3.* SOLR_VERSION=8.1.1 SOLR_CLOUD=false
+  - SYMFONY_VERSION=4.4.* SOLR_VERSION=8.2.0 SOLR_CLOUD=true
+  - SYMFONY_VERSION=4.3.* SOLR_VERSION=8.2.0 SOLR_CLOUD=false
   - SYMFONY_VERSION=4.3.* SOLR_VERSION=7.7.2 SOLR_CLOUD=true
-  - SYMFONY_VERSION=4.3.* SOLR_VERSION=7.7.2 SOLR_CLOUD=false
-  - SYMFONY_VERSION=4.3.* SOLR_VERSION=6.6.6 SOLR_CLOUD=false
+  - SYMFONY_VERSION=4.4.* SOLR_VERSION=7.7.2 SOLR_CLOUD=false
+  - SYMFONY_VERSION=3.4.* SOLR_VERSION=6.6.6 SOLR_CLOUD=false
```

Comment:
If we want to use another symfony version we should require http-foundation instead of dependency-injection.